indra/newview/llfloateravatar.cpp

```diff
@@ -40,16 +40,20 @@
 LLFloaterAvatar::LLFloaterAvatar(const LLSD& key)
 	:	LLFloater(key)
 {
+    mAvatarPicker = findChild<LLMediaCtrl>("avatar_picker_contents");
```

Comment:
This seems very wrong to me. findChild will happen before floater will have a chance to construct UI. This should happen in postBuild(), after xml was parsed.

Reply:
You're right, thanks.